src/pages/trips/Trips.tsx

```diff
@@ -1,37 +1,97 @@
 import * as React from 'react';
-import { Nav, NavItem, NavLink, Container, Col, Row } from 'reactstrap';
-import { Query, compose, graphql } from 'react-apollo';
+import { Nav, NavItem, NavLink, Container, Col, Row, Alert, Button, Fade } from 'reactstrap';
+import { Query } from 'react-apollo';
 import { Route, Redirect, Switch } from 'react-router';
 import { NavLink as RRNavLink } from 'react-router-dom';
+import { GET_GUEST_SORTED_BOOKINGS, Booking } from 'networking/bookings';
 
-import AudioLoading from 'shared/loading/AudioLoading';
+import { VIEWPORT_CENTER_LAYOUT } from 'styled/sharedClasses/layout';
 import NotFound from 'components/routes/NotFound';
-import { GET_GUEST_SORTED_BOOKINGS, Booking, Currency, GUEST_CANCEL_BOOKING } from 'networking/bookings';
-import { differenceInDays } from 'date-fns';
-import { cancel, loadWeb3 } from 'utils/web3';
+import Loading from 'components/shared/loading/Loading';
+import TripCard from 'components/work/TripCard';
+import { AlertProperties } from 'components/work/Alert/Alert';
+import ContactHostFormModal from 'components/work/ContactHostFormModal';
+import CancelBookingModal from 'components/work/CancelBookingModal.tsx';
+
+
+enum ModalType {
+  CANCEL_BOOKING = 'CANCEL_BOOKING',
+  CONTACT_HOST = 'CONTACT_HOST',
+}
+
+function Trips() {
+  const [modal, setModal] = React.useState<ModalType | undefined>(undefined);
+  const [alert, setAlert] = React.useState<AlertProperties>({ color: '', msg: '', show: false });
+  const [booking, setBooking] = React.useState<Booking | undefined>(undefined);
 
-const Trips = () => {
   return (
     <Query query={GET_GUEST_SORTED_BOOKINGS}>
       {({ loading, error, data }) => {
         if (loading) {
-          return <AudioLoading height={48} width={96} />;
+          return (
+            <Container tag={Fade} className={VIEWPORT_CENTER_LAYOUT}>
+              <Loading height="8rem" width="8rem" />
+            </Container>
+          );
         }
         if (error || !data) {
           return <h1>{error ? error.message : 'Error / No Data'}</h1>;
         }
 
-        const { cancelled, current, past, upcoming } = data;
+        const { current, upcoming } = data;
         const isCurrentEmpty = !(current || []).length;
-        // const isUpcomingEmpty = !(upcoming || []).length;
+        const isUpcomingEmpty = !(upcoming || []).length;
+        const renderCurrentEmpty = 
+          <>
+            <Row className="mb-2">
+              <h2>You have no trips awaiting approval.</h2>
+            </Row>
+            <Row>
+              <a href='/work'>
+                <Button>Book a Home Today!</Button>
```

Comment:
this can be `<a className="btn">` so the cursor is correctly registered.

Reply:
👍